extra/NSFW/src/predict.ts

```diff
@@ -0,0 +1,273 @@
+import * as fs from "fs";
+import * as request from "request";
+import * as tf from "@tensorflow/tfjs-node";
+import { Canvas, loadImage } from "canvas";
+import { download_image } from "../functions/functions";
+import { canvas } from "./interface";
+
+const options:any = { // options
```

Comment:
```suggestion
const options = {
```
Is there a reason why `options` should have the `any`-type?

Reply:
TSLint being a pain 😅